mycli/main.py

```diff
@@ -701,15 +701,43 @@ def output(self, output, status=None):
 
         """
         if output:
-            self.log_output(output)
-            special.write_tee(output)
-            special.write_once(output)
+            size = self.cli.output.get_size()
+
+            margin = self.get_output_margin(status)
+
+            fits = True
+            buf = []
+            output_via_pager = self.explicit_pager and special.is_pager_enabled()
+            for i, line in enumerate(output, 1):
+                self.log_output(line)
+                # line needs to be casted to unicode for click with Python 2.7
+                special.write_tee(text_type(line))
```

Comment:
We need to fix whatever is not returning unicode. Do you know what it is? I know the CSV part of CLI Helpers wasn't at one point, but that should be fixed in the version that mycli requires.

If we can't fix it, then, we'll need to convert to unicode for the call to `write_once()` as well (line 715).